src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VQSLODTranche.java

```diff
@@ -130,12 +130,12 @@ protected static VQSLODTranche emptyTranche(final List<VariantDatum> data, final
         return tranches;
     }
 
-    public static List<TruthSensitivityTranche> mergeAndConvertTranches(final Map<Double, List<VQSLODTranche>> scatteredTranches, final List<Double> tsLevels, final VariantRecalibratorArgumentCollection.Mode mode) {
+    public static List<TruthSensitivityTranche> mergeAndConvertTranches(final TreeMap<Double, List<VQSLODTranche>> scatteredTranches, final List<Double> tsLevels, final VariantRecalibratorArgumentCollection.Mode mode) {
         List<VQSLODTranche> mergedTranches = new ArrayList<>();
         List<TruthSensitivityTranche> gatheredTranches = new ArrayList<>();
 
         //make a list of merged tranches of the same length
-        for (final Double VQSLODlevel : scatteredTranches.keySet()) {
+        for (final Double VQSLODlevel : scatteredTranches.descendingKeySet()) {
```

Comment:
On line 145 I posited the assumption that the tranches were sorted, but they weren't because they originally went into a HashMap.  Mea culpa.